### PR TITLE
[MIG] sale_mrp: nothing to do

### PR DESCRIPTION
**Pull request by cubells**
Depends on:

- [x] mrp: #1076

cc @Tecnativa

### CODE REVIEW

File: odoo/openupgrade/doc/source/modules90-100.rst
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules90-100.rst]: Nothing to do

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Check Travis status to see if you have to close/reopen the PR to make it green

**Comment by cubells** [comment]
Travis error is no related to this pr, isn't it?

**Comment by pedrobaeza** [comment]
I think it is, as main branch is green and other PR in the same branch is as well.

**Comment by cubells** [comment]
mrp migration script fails. ??

**Comment by pedrobaeza** [comment]
The problem here is that mrp is still not migrated, so we can't merge this until the other PR is merged.

**Review by pedrobaeza — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
@cubells perhaps you may want to check https://github.com/OCA/OpenUpgrade/pull/1076 😏

**Comment by MiquelRForgeFlow** [comment]
Rebase :)

**Comment by cubells** [comment]
@mreficent done!

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**
